addons/rose/index.js

```diff
@@ -12,11 +12,43 @@ module.exports = {
     app.import('node_modules/codemirror/lib/codemirror.css');
     app.import('node_modules/codemirror/theme/monokai.css');
 
+    this.includeHDSStyles(app);
     this.includeFlightIcons(app);
     this.includePublic(app);
     this.setupSVGO(app);
   },
 
+  /**
+   * Due to a limitation in how ember treats nested addons (see https://github.com/ember-cli/ember-cli/issues/4475)
+   * this is neeeded to reach down into @hashicorp/ember-flight-icons' contentFor hook to run the logic
+   * that injects the sprite into the DOM
+   */
+  contentFor(type, config) {
+    return this.findOwnAddonByName('@hashicorp/design-system-components')
+      .findOwnAddonByName('@hashicorp/ember-flight-icons')
```

Comment:
This is awesome! Didn't know we could chain this which removes the extra import of flight icons

Reply:
I just discovered this.  One less direct dependency!